File: core/src/main/scala/ai/h2o/sparkling/backend/api/scalainterpreter/ScalaCodeResultV3.java
package ai.h2o.sparkling.backend.api.scalainterpreter;

import water.Iced;
import water.Key;
import water.api.schemas3.KeyV3;

public class ScalaCodeResultV3 extends KeyV3<Iced, ScalaCodeResultV3, ScalaCodeResult> {
    public ScalaCodeResultV3() {
    }

    public ScalaCodeResultV3(Key<ScalaCodeResult> key) {
        super(key);
    }
}

File: extensions/src/main/scala/ai/h2o/sparkling/extensions/rest/api/schema/ScalaCodeResultV3.scala
package ai.h2o.sparkling.extensions.rest.api.schema

import water.{Iced, Key}
import water.api.schemas3.KeyV3

class ScalaCodeResultV3(key: Key[ScalaCodeResult])
  extends KeyV3[Iced[ScalaCodeResult], ScalaCodeResultV3, ScalaCodeResult](key) {
  def this() = this(null)
}
